docs/source/markdown/podman-artifact-ls.1.md.in

```diff
@@ -43,11 +44,20 @@ quay.io/artifact/foobar2  special     cd734b558ceb       12.58MB
 
 List artifacts in the local store without truncating the digest
 ```
+$ podman artifact ls --no-trunc
 REPOSITORY                TAG         DIGEST                                                              SIZE
 quay.io/artifact/foobar1  latest      ab609fad386df1433f461b0643d9cf575560baf633809dcc9c190da6cc3a3c29    2.097GB
 quay.io/artifact/foobar2  special     cd734b558ceb8ccc0281ca76530e1dea1eb479407d3163f75fb601bffb6f73d0    12.58MB
 ```
 
+List artifacts in the local store without the title header
+```
+$ podman artifact ls --noheading
+REPOSITORY                TAG         DIGEST             SIZE
```

Comment:
That does not look right

Reply:
correct.  there was actually a mistake in the documentation and i was preoccupied with fixing that.  duh.